tests/test_singlepoint_cli.py

```diff
@@ -128,29 +158,87 @@ def test_singlepoint_calc_kwargs(tmp_path):
             results_path,
             "--property",
             "energy",
+            "--log",
+            log_path,
+            "--summary",
+            summary_path,
         ],
     )
     assert result.exit_code == 0
     assert "Using float32 for MACECalculator" in result.stdout
 
 
-def test_singlepoint_log(tmp_path, caplog):
+def test_log(tmp_path):
     """Test log correctly written for singlepoint."""
     results_path = tmp_path / "NaCl-results.xyz"
-    with caplog.at_level("INFO", logger="janus_core.single_point"):
-        result = runner.invoke(
-            app,
-            [
-                "singlepoint",
-                "--struct",
-                DATA_PATH / "NaCl.cif",
-                "--property",
-                "energy",
-                "--out",
-                results_path,
-                "--log",
-                f"{tmp_path}/test.log",
-            ],
-        )
-        assert "Starting single point calculation" in caplog.text
-        assert result.exit_code == 0
+    log_path = tmp_path / "test.log"
+    summary_path = tmp_path / "summary.yml"
+
+    result = runner.invoke(
+        app,
+        [
+            "singlepoint",
+            "--struct",
+            DATA_PATH / "NaCl.cif",
+            "--out",
+            results_path,
+            "--property",
+            "energy",
+            "--log",
+            log_path,
+            "--summary",
+            summary_path,
+        ],
+    )
+    assert result.exit_code == 0
+
+    # Read log file
+    with open(log_path, encoding="utf8") as log_file:
+        logs = yaml.safe_load(log_file)
+
+    # Check for correct messages anywhere in logs
+    messages = ""
+    for log in logs:
+        messages += log["message"]
```

Comment:
Honestly, I'm thinking we probably just want to stick this in a `utilities` function somewhere.